tracing-subscriber/src/fmt/time/chrono_crate.rs

```diff
@@ -0,0 +1,57 @@
+use crate::fmt::format::Writer;
+use crate::fmt::time::FormatTime;
+
+/// Formats [local time]s and [UTC time]s with [formatter] implementations
+/// that use the [`chrono` crate].
+///
+/// [local time]: https://docs.rs/chrono/0.4.26/chrono/offset/struct.Local.html
+/// [UTC time]: https://docs.rs/chrono/0.4.26/chrono/offset/struct.Utc.html
+/// [`chrono` crate]: https://docs.rs/chrono/0.4.26/chrono/
```

Comment:
let's make use of intra-doc links instead:

```suggestion
/// [local time]: [`chrono::offset::Local`]
/// [UTC time]: [`chrono::offset::Utc`]
/// [`chrono` crate]: [`chrono`]
```

(note that I haven't tried this locally, but it _should_ work)